core/controllers/access_validators_test.py

```diff
@@ -469,3 +471,82 @@ def test_validation_returns_false_if_given_user_is_non_existent(
             ), expected_status_int=404
         )
         self.logout()
+
+
+class TopicEditorPageAccessValidationHandlerTests(test_utils.EmailTestBase):
```

Comment:
Why is this using EmailTestBase and not the generic one?

Reply:
The test i removed from the `topic_editor_test` file was using the `EmailTestBase` that's why i used it here too, Updated it to `GenericTestBase`